Added husky for pre-commit hooks

njs202 opened the pull request:
(fixes #58)

ntsummers1 commented:
Nice work! Since we talked about this, I'll just go over the 2 slight changes here. But this is dope.

1. I think pre-push is ok to get rid of for now since the git action does a check on pull request creation and the git hooks are so simple. With tests and then pushing out to a beta site automatically we might want but for now just pre-commits are fine

2. I want to make sure we keep a slight running doc of what we add to the repo so students and staff can get the project running and can create a commit day one. Can you add a quick note about this command in the readme file? I think for a student who has no idea what a hook is might find it helpful.

njs202 commented:
> Nice work! Since we talked about this, I'll just go over the 2 slight changes here. But this is dope.
> 
> 1. I think pre-push is ok to get rid of for now since the git action does a check on pull request creation and the git hooks are so simple. With tests and then pushing out to a beta site automatically we might want but for now just pre-commits are fine
> 2. I want to make sure we keep a slight running doc of what we add to the repo so students and staff can get the project running and can create a commit day one. Can you add a quick note about this command in the readme file? I think for a student who has no idea what a hook is might find it helpful.

1. Removed

2. Added

njs202 commented:
Guess I should close it after merging the branch

ntsummers1 reviewed (approved):
Nice. Feel free to hit the old squash and merge button at the bottom to close this whenever you're ready!
